test/proposals/mips/mip-b05/mip-b05.sol

```diff
@@ -0,0 +1,314 @@
+                addresses.getAddress("JUMP_RATE_IRM_MOONWELL_WETH")
+            ),
+            "Set interest rate model for Moonwell WETH to updated rate model"
+        );
+
+        IRParams memory previousStablecoinIRParams = IRParams({
+            baseRatePerTimestamp: 0,
+            kink: 0.8e18,
+            multiplierPerTimestamp: 0.05e18,
+            jumpMultiplierPerTimestamp: 2.5e18
+        });
+
+        // =========== DAI IR Update ============
+
+        // Check Preconditions
+        address daiPreviousJumpRateModelAddress = 0xbc93DdFAE192926BE036c6A6Dd544a0e250Ab97D;
+        _validateJRM(
+            daiPreviousJumpRateModelAddress,
+            addresses.getAddress("MOONWELL_DAI"),
+            previousStablecoinIRParams
+        );
+
+        // Add update action
+        _pushCrossChainAction(
+            addresses.getAddress("MOONWELL_DAI"),
+            abi.encodeWithSignature(
+                "_setInterestRateModel(address)",
+                addresses.getAddress("JUMP_RATE_IRM_MOONWELL_DAI")
+            ),
+            "Set interest rate model for Moonwell DAI to updated rate model"
+        );
+
+        // =========== USDC IR Update ============
+
+        // Check Preconditions
+        address usdcPreviousJumpRateModelAddress = 0x93AEE5b2431991eB96869057e98B0a7e9262cDEb;
+        _validateJRM(
+            usdcPreviousJumpRateModelAddress,
+            addresses.getAddress("MOONWELL_USDC"),
+            previousStablecoinIRParams
+        );
+
+        // Add update action
+        _pushCrossChainAction(
+            addresses.getAddress("MOONWELL_USDC"),
+            abi.encodeWithSignature(
+                "_setInterestRateModel(address)",
+                addresses.getAddress("JUMP_RATE_IRM_MOONWELL_USDC")
+            ),
+            "Set interest rate model for Moonwell USDC to updated rate model"
+        );
+
+        // =========== USDbC IR Update ============
+
+        // Check Preconditions
+        address usdbcPreviousJumpRateModelAddress = 0x1603178b26C3bc2cd321e9A64644ab62643d138B;
+        _validateJRM(
+            usdbcPreviousJumpRateModelAddress,
+            addresses.getAddress("MOONWELL_USDBC"),
+            previousStablecoinIRParams
+        );
+
+        // Add update action
+        _pushCrossChainAction(
+            addresses.getAddress("MOONWELL_USDBC"),
+            abi.encodeWithSignature(
+                "_setInterestRateModel(address)",
+                addresses.getAddress("JUMP_RATE_IRM_MOONWELL_USDBC")
+            ),
+            "Set interest rate model for Moonwell USDbC to updated rate model"
+        );
+    }
+
+    function run(Addresses addresses, address) public override {
+        _simulateCrossChainActions(addresses.getAddress("TEMPORAL_GOVERNOR"));
+    }
+
+    function printCalldata(Addresses addresses) public override {
+        printActions(
+            addresses.getAddress("TEMPORAL_GOVERNOR"),
+            addresses.getAddress("WORMHOLE_CORE")
```

Comment:
```suggestion
            addresses.getAddress(
                "WORMHOLE_CORE",
                sendingChainIdToReceivingChainId[block.chainid]
            )
```

Reply:
this line caused the calldata printed to be incorrect as it retrieved the wormhole core address on base, and not the address on moonbeam where the message needed to be published.